hadoop-ozone/dist/src/main/smoketest/s3/MultipartUpload.robot

```diff
@@ -148,6 +148,15 @@ Test Multipart Upload Complete
     ${result} =                 Execute AWSS3ApiCli        get-object --bucket ${BUCKET} --key ${PREFIX}/multipartKey1 --part-number 2 /tmp/${PREFIX}-multipartKey1-part2.result
     Compare files               /tmp/part2        /tmp/${PREFIX}-multipartKey1-part2.result
 
+#check whether parts count is retrieved through get and head object
+    ${result}                   Execute AWSS3ApiCli        get-object --bucket ${BUCKET} --key ${PREFIX}/multipartKey1 --part-number 1 /tmp/${PREFIX}-multipartKey1-part1.result
+    ${partsCount}               Execute and checkrc        echo '${result}' | jq -r '.PartsCount'    0
+                                Should Be Equal            ${partsCount}    2
```

Comment:
To verify different items in the response for the same multipart key, we shouldn't execute `get-object` again.  We should save the output (`${result}`) in a temp file and perform assertions on it.

Example:

https://github.com/apache/ozone/blob/0d8d1522a7a6d67dab6572da46cd72a148e6d052/hadoop-ozone/dist/src/main/smoketest/admincli/datanode.robot#L33-L42

Reply:
Or in this PR, since the statements are part of the same big test case, we can directly check `${result}` without using a temporary file.